eng/run-test-job.yml

```diff
@@ -69,13 +86,45 @@ jobs:
 
     steps:
 
-    # Download and unzip tests    
+    # Download and unzip managed test artifacts
     - template: /eng/download-artifact-step.yml
       parameters:
         downloadFolder: $(binTestsPath)
         unpackFolder: $(testRootFolderPath)
-        artifactFileName: $(osGroup).$(archType).$(buildConfigUpper)$(archiveExtension)
+        artifactFileName: ${{ parameters.managedBuildOsIdentifier }}.$(archType).$(buildConfigUpper)$(archiveExtension)
         artifactName: $(testArtifactName)
+        displayName: 'managed test artifacts (built on ${{ parameters.managedBuildOsIdentifier }})'
+
+
+    # Download and unzip native test artifacts into the obj folder
+    - template: /eng/download-artifact-step.yml
+      parameters:
+        downloadFolder: $(binTestsPath)/tmp
+        unpackFolder: $(binTestsPath)/obj/$(osGroup).$(archType).$(buildConfig)/Native/src
+        cleanUnpackFolder: true
+        artifactFileName: 'NativeTestComponents-$(osIdentifier)-$(archType)-$(buildConfig)$(archiveExtension)'
+        artifactName: 'NativeTestComponents-$(osIdentifier)-$(archType)-$(buildConfig)'
+        displayName: 'native test artifacts'
+
+
+    # Copy native components to the test output folder
+    - ${{ if ne(parameters.osGroup, 'Windows_NT') }}:
+      - script: ./build-test.sh copynativeonly skipgeneratelayout $(buildConfig) $(archType) $(priorityArg) ci
+        displayName: Copy native components to test output folder
+    - ${{ if eq(parameters.osGroup, 'Windows_NT') }}:
+      - script: build-test.cmd skipnative skipgeneratelayout $(buildConfig) $(archType) $(priorityArg) ci
```

Comment:
```suggestion
      - script: build-test.cmd copynativeonly skipgeneratelayout $(buildConfig) $(archType) $(priorityArg) ci
```

Reply:
Ahh, damn copy-paste, thanks for spotting!